### PR TITLE
Configured Gradle Play Publisher Plugin

### PR DESCRIPTION
**Pull request by AvijitGhosh82**

### CODE REVIEW

**Comment by AvijitGhosh82** [comment]
@rkorytkowski this looks fine to me, but the build fails: https://travis-ci.org/AvijitGhosh82/openmrs-contrib-android-client/builds/135863906

Could you tell me what's happening? I have followed whatever the plugin docs say.

File: build.sh [hunk 1]
```diff
@@ -16,7 +17,10 @@ if [ -n "$TRAVIS_TAG" ]; then
 
 echo "Building relase apk for Github and Play Store..."
 echo $KEYSTORE_GPG_PASSPHRASE | gpg --passphrase-fd 0 release.keystore.gpg
+echo $KEYSTORE_GPG_PASSPHRASE | gpg --passphrase-fd 0 google_play.json.gpg
```
Comment (robbyoconnor) [inline build.sh]: Are you freaking kidding me? THIS IS BAD.

File: build.sh [hunk 2]
```diff
@@ -16,7 +17,10 @@ if [ -n "$TRAVIS_TAG" ]; then
...
+
```
Comment (robbyoconnor) [inline build.sh]: Nevermind.

**Comment by AvijitGhosh82** [comment]
Update: @robbyoconnor The JSON Key file that had been given does not have the "type" field. That is causing the problem.

**Comment by AvijitGhosh82** [comment]
@rkorytkowski Everything works now :) Please check the console, you'll find a new APK in the Alpha track. I also added 1000 to the travis build number so that the release build has a higher number than the alpha build. I think this is ready to go.

File: openmrs-client/build.gradle [hunk 3]
```diff
@@ -1,13 +1,28 @@
...
+
 android {
     compileSdkVersion 23
     buildToolsVersion "23.0.2"
     useLibrary 'org.apache.http.legacy'
 
     def travisBuildNumber = System.getenv("TRAVIS_BUILD_NUMBER")
-    def buildNumber = (travisBuildNumber) ? (travisBuildNumber as int) : 1
-	
+    def buildNumber = (travisBuildNumber) ? ((travisBuildNumber as int)+1000) : 1
```
Comment (rkorytkowski) [inline openmrs-client/build.gradle]: Please put 1000 in a variable called lastBuildNumberPriorToTravis and add a comment that it was the last build number released to play store before introducing travis build numbers. Was it really 1000 or less?

**Comment by rkorytkowski** [comment]
Could you please not commit .DS_Store files? Maybe add them to gitignore?

**Comment by robbyoconnor** [comment]
> Could you please not commit .DS_Store files? Maybe add them to gitignore?

These should always (as a matter of practice) be removed **_ALWAYS_**

**Pull request merged**